packages/flutter/lib/src/rendering/viewport.dart

```diff
@@ -108,11 +108,21 @@ abstract interface class RenderAbstractViewport extends RenderObject {
   /// when the offset of the viewport is changed by x then `target` also moves
   /// by x within the viewport.
   ///
+  /// The optional [axisDirection] is used by [RenderTwoDimensionalViewport] to
+  /// determine which [Axis] of the two should reveal. One dimensional
+  /// subclasses like [RenderViewportBase and [RenderListWheelViewport] will
+  /// assert that if an [axisDirection] is provided, that it matches the
+  /// [axisDirection] the viewport has been configured for.
+  ///
```

Comment:
I am wondering how this change plays into implicit accessibility scrolling. This method is basically the workhorse behind implicit a11y scrolling powering `RenderViewportBase.showInViewport`, which is called from `RenderObject.showOnScreen`.

(Implicit a11y scrolling is if you move the a11y focus beyond the edge of a scrollable and it then automatically reveals the item that has gotten a11y focus.)